Meta/run.sh

```diff
@@ -43,9 +49,14 @@ SERENITY_RUN="${SERENITY_RUN:-$1}"
     fi
 }
 
+[ -e "$SERENITY_QEMU_BUILD_DIR/$ARCH/qemu/qemu-system-$SERENITY_QEMU_TARGET_ARCH" ] && SERENITY_QEMU_BIN="$SERENITY_QEMU_BUILD_DIR/$ARCH/qemu/qemu-system-$SERENITY_QEMU_TARGET_ARCH"
+
 SERENITY_QEMU_MIN_REQ_VERSION=5
 installed_major_version=$("$SERENITY_QEMU_BIN" -version | head -n 1 | grep -Po "(?<=QEMU emulator version )([1-9]\d*|0)")
-[ "$installed_major_version" -lt "$SERENITY_QEMU_MIN_REQ_VERSION" ] && die "Required QEMU >= 5.0! Found $($SERENITY_QEMU_BIN -version | head -n 1)"
+if [ "$installed_major_version" -lt "$SERENITY_QEMU_MIN_REQ_VERSION" ]; then
+    "$TOOLCHAIN/BuildQemu.sh" || die "Could not install QEMU"
```

Comment:
This might not be a useful task for the user if their qemu binary is a native Windows Qemu they are using in /mnt/c/Program Files/qemu. A qemu built for WSL2 will not be able to use any hardware acceleration (until a future windows release enables nested virtualization). I think a prompt or log message that the user must install the new qemu is the most portable? Maybe a check to see if the user's qemu ends in ".exe"?

Reply:
Thanks! To be honest, I didn't even think about Windows users which was kind of a stupid oversight.

> Maybe a check to see if the user's qemu ends in ".exe"?

Are you implying that if it doesn't end in ".exe", only then should we execute this? Or, are you implying that this is not at all a good idea and we should discard the patch?

Reply:
I'm saying that if we do want to have the auto install qemu feature, it should avoid doing so if the SERENITY_QEMU_BIN is a windows executable. 

If other folks don't like how automated it is, changing the error message to point to the Toolchain/BuildQEMU.sh script might be a good alternative? I don't really mind either way myself.